airflow/contrib/operators/cassandra_to_gcs.py

```diff
@@ -41,6 +41,35 @@ class CassandraToGoogleCloudStorageOperator(BaseOperator):
     Copy data from Cassandra to Google cloud storage in JSON format
 
     Note: Arrays of arrays are not supported.
+
+    :param cql: The CQL to execute on the Cassandra table.
+    :type cql: str
+    :param bucket: The bucket to upload to.
+    :type bucket: str
+    :param filename: The filename to use as the object name when uploading
+        to Google cloud storage. A {} should be specified in the filename
+        to allow the operator to inject file numbers in cases where the
+        file is split due to size.
+    :type filename: str
+    :param schema_filename: If set, the filename to use as the object name
+        when uploading a .json file containing the BigQuery schema fields
+        for the table that was dumped from MySQL.
+    :type schema_filename: str
+    :param approx_max_file_size_bytes: This operator supports the ability
+        to split large table dumps into multiple files (see notes in the
+        filenamed param docs above). Google cloud storage allows for files
+        to be a maximum of 4GB. This param allows developers to specify the
```

Comment:
As mentioned in another PR.
This is wrong. The limit is not 4GB it's 5GB:
https://cloud.google.com/storage/quotas
"There is a maximum size limit of 5 TB for individual objects stored in Cloud Storage."
I think in general It's best not to mention it at all. The limits can change at will. Airflow can't keep track of all size changes.

Reply:
You are right, this needs to be modified in all GCS related Operators, I will create a PR to remove this lines everywhere and add a reference to `https://cloud.google.com/storage/quotas` page